src/include/ceph_features.h

```diff
@@ -248,6 +249,7 @@ DEFINE_CEPH_FEATURE_DEPRECATED(63, 1, RESERVED_BROKEN, LUMINOUS) // client-facin
 	 CEPH_FEATURE_SERVER_NAUTILUS |		\
 	 CEPH_FEATURE_CEPHX_V2 | \
 	 CEPH_FEATURE_OSD_PGLOG_HARDLIMIT | \
+	 CEPH_FEATURE_MDS_USER_GROUP_QUOTA | \
```

Comment:
add feature bit to src/mds/cephfs_features.h

Reply:
fixed it ,thanks.